content/api/metrics.apib

```diff
@@ -7,62 +7,69 @@ description: Query real-time data about message processing, message disposition,
+| `count_block_bounce`             | Total number of Bounced messages due to an IP block                                                 |
+| `count_admin_bounce`             | Total number of Bounced messages due to admin bounce classification reasons, also includes Rejected |
+| `count_undetermined_bounce`      | Total number of Bounced messages due to undetermined bounce reasons                                 |
+| `count_rejected`                 | Messages rejected due to policy or that failed to generate                                          |
+| `count_policy_rejection`         | Messages rejected by SparkPost due to policy                                                        |
+| `count_generation_failed`        | Message generation failed for an intended recipient                                                 |
+| `count_generation_rejection`     | Messages rejected by SparkPost due to policy                                                        |
+| `count_inband_bounce`            | Messages that bounced on delivery attempt during the SMTP session                                   |
+| `count_outofband_bounce`         | Messages that the ISP bounced subsequent to a successful delivery                                   |
+| `count_delayed`                  | Total number of delays due to any temporary failure                                                 |
+| `count_delayed_first`            | Messages delayed on the first delivery attempt                                                      |
+| `total_msg_volume`               | Total size of delivered messages, in bytes (including attachments)                                  |
+| `count_spam_complaint`           | Number of spam complaints received from an ISP                                                      |
+| `total_delivery_time_first`      | Total time taken to deliver messages on first attempt (milliseconds)                                |
+| `total_delivery_time_subsequent` | Total time taken to delivery messages on subsequent attempts (milliseconds)                         |
+| `count_unsubscribe`              | Total number of unsubscribes as a result of clicked links and the ISP list unsubscribe feature      |
+
+
+<banner status="info"><strong>Note</strong>: For a given request, average first attempt delivery latency can be calculated as <code>total_delivery_time_first / count_delivered</code>. A similar calculation holds for <code>total_delivery_time_subsequent</code>.</banner>
+
+
+### Discoverability Links [GET /metrics/]
+
+The Metrics API is designed for discoverability of child links. Calling the API root displays a list of URIs that exists within the Metrics API.
+
+<banner status="info"><strong><a href="https://www.sparkpost.com/enterprise-email/">SparkPost Enterprise customers only</a></strong>: Metric discoverability links for binding-groups and bindings apply to SparkPost Enterprise customers only.</banner>
```

Comment:
Did we reach any conclusion on whether Binding Groups are at all still in use in Enterprise?

Reply:
It is a case-by-case basis so I'm going to leave this along for now.